www/js/input-page.js

```diff
@@ -537,17 +538,31 @@ var submit = function( ) {
 			return;
 	}
 
-	if(!leafPhotoTaken){
-		navigator.notification.alert("Please take a leaf photo.");
-		return;
-	}
-	else{
-		leafImageURI = $("#leaf-photo").prop("src");
-	}
+	//if(!leafPhotoTaken){
+	//	navigator.notification.alert("Please take a leaf photo.");
+	//	return;
+	//}
+	//else{
+	//	leafImageURI = $("#leaf-photo").prop("src");
+	//}
 	//Check validity of site password
 	//Attempt to submit survey if password is valid
 	//navigator.notification.alert("SiteID: " + siteID +
 	//	"\nSite password: " +sitePassword);
+	var online = navigator.onLine;
+	if(online == false){
+		db.transaction(function(tx){
+                        tx.executeSql("INSERT INTO SURVEY VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)", ['survey',siteID,getURLParameter("userID"),getURLParameter("password"),circle,survey,dateTime,temperatures[temperature].min,temperatures[temperature].max,$(".notes").val(),plantSpecies,herbivoryValue,surveyType,parseInt(leafCount),"Mobile"]);
+                    }  , function(error){
+                        alert("Transaction Error: "+error.message);
+                    },function(){
+						alert("This page was successfully stored");
+						window.location = "homepage.html";
```

Comment:
This part is actually wrong, the homepage url should include userId and pw. Checkout the login procedure I had. You have the userid and pw already, just add them as part of the url.